react-client/src/utils/domUtils.js

```diff
@@ -98,11 +109,12 @@ function extractDomInfo(elTarget) {
   );
   
   const elParent = elTarget.parentElement;
+  
   const parent = {
     id: elParent.id && `#${elParent.id.trim()}`,
-    tag: elParent.localName,
-    class: elParent.className && `.${elParent.className.trim()}`,
-    classes: [...elParent.classList]
+    tag: elParent.localName,    
+    class: elParent.className && `.${elParent.className}`,
+    classes: [...elParent.classList.value.split(" ").filter(customFilter => !customFilter.includes('custom-css'))]    
```

Comment:
Not sure why we need to use split here when classList is already converted as array type.

Can't we do the following instead:

`[...elParent.classList].filter(customFilter => !customFilter.includes('custom-css'))`

Reply:
we need .split() because it is a domtokenlist not an array filter is not an available option in domtokenlist